src/coreclr/nativeaot/Runtime/RhConfig.cpp

```diff
@@ -179,6 +191,103 @@ void RhConfig::ReadEmbeddedSettings()
     return;
 }
 
+bool RhConfig::GetEmbeddedKnob(_In_z_ const char* configName, _Out_ const char** configValue)
+{
+    // Read the config if we haven't yet
+    if (g_embeddedKnobs == NULL)
+    {
+        ReadEmbeddedKnobs();
+    }
+
+    // Config wasn't read or reading failed
+    if (g_embeddedKnobs == CONFIG_INI_NOT_AVAIL)
+    {
+        return false;
+    }
+
+    const ConfigPair* configPairs = (const ConfigPair*)g_embeddedKnobs;
+
+    // Find the first name which matches (case insensitive to be compat with environment variable counterpart)
+    for (int iSettings = 0; iSettings < RCV_Count; iSettings++)
```

Comment:
`RCV_Count` only make sense if we know the upper bound because this is not something that is user-configurable. RuntimeHostConfigurationOption is unbounded. We need to keep track of the real count. (Same comment applies to other uses of RCV_Count in this file)